OpenSim/Simulation/SimbodyEngine/SimbodyEngine.h

```diff
@@ -185,6 +162,53 @@ OpenSim_DECLARE_CONCRETE_OBJECT(SimbodyEngine, Object);
     void convertQuaternionsToDirectionCosines(double aQ1, double aQ2, double aQ3, double aQ4, double rDirCos[3][3]) const;
     void convertQuaternionsToDirectionCosines(double aQ1, double aQ2, double aQ3, double aQ4, double *rDirCos) const;
 
+    //-------------------------------------------------------------------------
+    // DEPRECATED METHODS
+    //-------------------------------------------------------------------------
+    /** @name Deprecated */
+    // @{
+    /** <b>(Deprecated)</b> Use Frame::findLocationInGround() instead. */
+    DEPRECATED_14("use Frame::findLocationInGround() instead")
+    void getPosition(const SimTK::State& s, const PhysicalFrame &aBody, const SimTK::Vec3& aPoint, SimTK::Vec3& rPos) const;
+    DEPRECATED_14("use Frame::findVelocityInGround() instead")
+    void getVelocity(const SimTK::State& s, const PhysicalFrame &aBody, const SimTK::Vec3& aPoint, SimTK::Vec3& rVel) const;
+    DEPRECATED_14("use Frame::findAccelerationInGround() instead")
+    void getAcceleration(const SimTK::State& s, const PhysicalFrame &aBody, const SimTK::Vec3& aPoint, SimTK::Vec3& rAcc) const;
+    DEPRECATED_14("use Frame::getTransformInGround().R() instead")
+    void getDirectionCosines(const SimTK::State& s, const PhysicalFrame &aBody, double rDirCos[3][3]) const;
+    DEPRECATED_14("use Frame::getTransformInGround().R() instead")
+    void getDirectionCosines(const SimTK::State& s, const PhysicalFrame &aBody, double *rDirCos) const;
+    DEPRECATED_14("use Frame::getVelocityInGround()(0) instead")
+    void getAngularVelocity(const SimTK::State& s, const PhysicalFrame &aBody, SimTK::Vec3& rAngVel) const;
+    DEPRECATED_14("use Frame::getVelocityInGround()(0) instead")
```

Comment:
I'm confused why two methods with different names would be returning the same thing. I would expect a user to call `getAngularVelocity()` if they wanted it relative to Ground. For `getAngularVelocityBodyLocal()`, I suggest using the comment `"see Frame::getVelocityInGround()(0)"` on L183 so a user updating old code will (hopefully) check that the code is doing the right thing rather than blindly replacing with the new method.